src/content/en/2020/markup.md

```diff
@@ -543,58 +607,64 @@ When we built our tests we didn’t check for the presence of images, but only l
+| `<button type="reset">` | 41,735 | 0.49% |
 
-<figcaption>{{ figure_link(caption="Top 7 button setups.", sheets_gid="410549982", sql_file="pages_markup_by_device.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Adoption of button types.", sheets_gid="410549982", sql_file="pages_markup_by_device.sql") }}</figcaption>
 </figure>
 
-According to the numbers we gathered, about 60% of the pages contain a button element and more than half of them (32.43%) fail to specify a `type` attribute.
-
-<div class="note">
-  <p>The button element has a <a href="https://dev.w3.org/html5/spec-LC/the-button-element.html">default type of <code>submit</code></a>, meaning that the default behavior of a button is to submit the current form data. For extra clarity, consider specifying the button type in order to avoid unexpected situations.</p>
-  <p>The following table shows that a page in the 90th percentile has at least 13 native buttons, while the pages in the 10th and 25th percentiles contain no buttons at all:</p>
-</div>
+Our analysis shows that about 60% of pages contain a button element and more than half of the pages (32.43%) have at least one button that fails to specify a `type` attribute. Note that the button element has a [default type](https://dev.w3.org/html5/spec-LC/the-button-element.html) of `submit`, so the default behavior of buttons on these 32% of pages is to submit the current form data. To avoid possibly unexpected behavior like this, a best practice is to specify the type attribute.
 
-<!-- Really part of the note above, but kept outside for simplicity: -->
 <figure markdown>
-| Percentile | Buttons |
+| Percentile | Buttons per page |
 |---|---|
+| 10 | 0 |
 | 25 | 0 |
 | 50 | 1 |
 | 75 | 5 |
 | 90 | 13 |
 
-<figcaption>{{ figure_link(caption="Button percentiles.", sheets_gid="309769153", sql_file="pages_markup_by_device_and_percentile.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Distribution of the number of buttons per page.", sheets_gid="309769153", sql_file="pages_markup_by_device_and_percentile.sql") }}</figcaption>
 </figure>
 
+Pages in the 10th and 25th percentiles contain no buttons at all, while a page in the 90th percentile contains 13 native `button` elements. In other words, 10% of pages contain 13 or more buttons.
+
 ### Link targets
 
-What protocols do anchors—`a` elements—point to? We looked at that information to identify the most popular protocols. How to read this information? Each row shows how many links with that protocol we count, and on how many of all documents at least one such protocol link is being used.
+The [anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a), or `a` element, links web resources together. In this section, we analyze the adoption of the protocols included in these link targets.
 
 <figure markdown>
-| Protocol | Number of pages referencing the protocol | Percentage |
+| Protocol | Occurrences | Pages (%) |
```

Comment:
General note (@rviscomi)—all those header changes look like thin ice to me. If these are changed they need scrutiny to make sure we don’t inadvertently change meaning. (We had some discussions related to this and had taken a lot of care during editing of the doc that the headers are right. I don’t have the time right now to check all those new header labels to verify they are still accurate—all I can say is that they’re a minefield and should not be changed lightly.)

Reply:
There were a few headings I didn't change for that reason, but it's worth a second look at the ones that I did change to make sure that they're correct.

Reply:
This certainly isn't occurrences, but pages.
```suggestion
| Protocol | Pages | Pages (%) |
```